pkg/api/server/register_networks.go

```diff
@@ -106,7 +111,7 @@ func (s *APIServer) registerNetworkHandlers(r *mux.Router) error {
 	//    required: true
 	//    description: the name of the network
 	//  - in: query
-	//    name: Force
+	//    name: force
```

Comment:
Nit (for cleaner potential cherry-picking): Can you break this change out into a separate commit?

Reply:
I wouldn't worry. We're past the point that anything should be going into 2.0 without a critical bug being behind it.